src/qml/options_model.h

```diff
@@ -35,27 +39,33 @@ class OptionsQmlModel : public QObject
     void setPrune(bool new_prune);
     int pruneSizeGB() const { return m_prune_size_gb; }
     void setPruneSizeGB(int new_prune_size);
+    int scriptThreads() const { return m_script_threads; }
+    void setScriptThreads(int new_script_threads);
     bool server() const { return m_server; }
     void setServer(bool new_server);
     bool upnp() const { return m_upnp; }
     void setUpnp(bool new_upnp);
 
 Q_SIGNALS:
+    void dbcacheSizeMiBChanged(int new_dbcache_size_mib);
     void listenChanged(bool new_listen);
     void natpmpChanged(bool new_natpmp);
     void pruneChanged(bool new_prune);
     void pruneSizeGBChanged(int new_prune_size_gb);
+    void scriptThreadsChanged(int new_script_threads);
     void serverChanged(bool new_server);
     void upnpChanged(bool new_upnp);
 
 private:
     interfaces::Node& m_node;
 
     // Properties that are exposed to QML.
+    bool m_dbcache_size_mib;
```

Comment:
```suggestion
    int m_dbcache_size_mib;
```

Reply:
Apparently clang-tidy can catch these issues. I was surprised gcc doesn't warn about this.

Reply:
hehe